site/en/docs/privacy-sandbox/user-agent/index.md

```diff
@@ -131,8 +129,7 @@ Vary: Device-Memory, Viewport-Width
 Critical-CH: Device-Memory
 ```
 
-If, after processing the `Accept-CH header`, the client would send a critical
-hint, the client retries the request.
+If the original request included a `Critical-CH`, after processing the `Accept-CH header`, the  client retries the request.
```

Comment:
Is this correct?

Reply:
```suggestion
If the browser requires a critical hint (`Critical-CH`) to properly render the webpage, then the server can ask for this additional information with the `Accept-CH` header. Then, the browser can send a new request for the page, including the critical hint.
```

I think this suggestion makes a little more sense.